backend/src/controllers/tour.js

```diff
@@ -73,8 +82,30 @@ const search = (req, res) => {
     //res.send(JSON.stringify(req.query));
     let query = {};
 
-    if (req.query.difficulty !== undefined) {
-        query.difficulty = req.query.difficulty;
+    if (req.query.difficulties !== undefined) {
+        let arrayDifficulties = req.query.difficulties.split(',');
+        //res.send(arrayDifficulties.length)
+        if (arrayDifficulties.length === 2) {
+            query.$or = [
+                {difficulty: arrayDifficulties[0]},
+                {difficulty: arrayDifficulties[1]}
+            ]
+        } else if (arrayDifficulties.length === 3) {
+            query.$or = [
+                {difficulty: arrayDifficulties[0]},
+                {difficulty: arrayDifficulties[1]},
+                {difficulty: arrayDifficulties[2]}
+            ]
+        } else if (arrayDifficulties.length === 4) {
+            query.$or = [
+                {difficulty: arrayDifficulties[0]},
+                {difficulty: arrayDifficulties[1]},
+                {difficulty: arrayDifficulties[2]},
+                {difficulty: arrayDifficulties[3]}
```

Comment:
This is very ugly way how to do that. To build the $or array you can use nice map function. arrayDifficulities.map((diff) => {return {difficulity: diff}});